cvat/apps/dataset_manager/tests/assets/annotations.json

```diff
@@ -43,6 +43,49 @@
       }
     ]
   },
+  "CVAT for video 1.1 attributes in tracks": {
+    "version": 0,
```

Comment:
Annotations don't follow the exact scenario which was described in the issue. Just want to check. Is it OK?

Reply:
These annotations also caught this bug. I changed it to the annotations from the issue description.